README.md

```diff
@@ -13,88 +13,103 @@ Some stuff is hard, and that is okay, but listing your kafka topics shouldn't be
 * Get detailed Overviews of Resources (Topics, Consumergroups, Brokers)
 * Create/Delete Topics
+* Edit Topic Configurations
+* Edit Consumer Offset for Topics
+* SASL/SSL Support support out of the box
+* Consume and Produce to and from Avro and Plaintext Topics (including Avro Schema Resolution from Schema Registry)
 * Context Switch (Easily Switch between pre-defined Clusters)
 * Kafka Ping (Test roundtrip time to your kafka cluster)
 
-## Installation and Usage
+## Command Overview
 
-### Installation
+```bash
+$ esque
+Usage: esque [OPTIONS] COMMAND [ARGS]...
 
-`esque` is available at pypi.org and can be installed with `pip install esque`. `esque` requires Python 3.6+ to run.
+  esque - an operational kafka tool.
 
-### Autocompletion
+Options:
+  --recreate-config  Overwrites the config with the sample config.
+  --version          Show the version and exit.
+  -v, --verbose
+  --no-verify        Skips all verification dialogs and answers them with yes.
+  --help             Show this message and exit.
 
-Autocompletion is automatically installed via a post-install hook in the setup.py. 
-If it doesn't work for some reason you can still install it yourself: 
+Commands:
+  apply     Apply a configuration
+  config    Configuration-related options
+  consume   Consume messages of a topic from one environment to a file or...
+  create    Create a new instance of a resource.
+  ctx       Switch clusters.
+  delete    Delete a resource.
+  describe  Get detailed information about a resource.
+  edit      Edit a resource
+  get       Get a quick overview of different resources.
+  ping      Tests the connection to the kafka cluster.
+  produce   Produce messages from <directory> based on output from transfer...
+```
 
-#### Bash
+## Installation and Usage
 
-```
- echo 'eval "$(_ESQUE_COMPLETE=source esque)"' >> ~/.esque/autocompletion.sh
- echo "source ~/.esque/autocompletion.sh" >> ~/.bashrc
-```
+### Installation
 
-#### ZSH
+`esque` is available at [pypi.org](https://pypi.org/project/esque/) and can be installed with `pip install esque`. `esque` requires Python 3.6+ to run.
 
-```
-echo 'eval "$(_ESQUE_COMPLETE=source_zsh esque)"' >> ~/.esque/autocompletion.zsh
-echo "source ~/.esque/autocompletion.zsh" >> ~/.zshrc
-```
+### Autocompletion
+
+The autocompletion scripts for `bash` and `zsh` can be generated by running `esque config autocomplete`.
 
 ### Usage
 
 #### Config Definition
 
 When starting `esque` for the first time the following message will appear:
 
-```
+```bash
 No config provided in ~/.esque
 Should a sample file be created in ~/.esque [y/N]:
 ```
 
-When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/sample_config.cfg) to `~/.esque/esque.cfg`.
+When answering with `y` `esque` will copy over the [sample config](https://github.com/real-digital/esque/blob/master/esque/config/sample_config.yaml) to `~/.esque/esque.yaml`.
```

Comment:
is it esque.yaml or esque_config.yaml now?

Reply:
I have an `esque.yaml` locally. :)